suggestions_and_questions.md

```diff
@@ -0,0 +1,7 @@
+-- **Suggestions and Questions** file --  Please edit to add your questions or suggestions to this file.
```

Comment:
Since this line would be a header I would suggest making it more like

```
# Suggestions and Questions file
Please edit to add your questions or suggestion to this file.
...
```

Then there can be different sections with different header levels, and there can be a list of items even.

Reply:
Sound good.